src/ProjectSystem/Environment.fs

```diff
@@ -63,13 +108,13 @@ module Environment =
 
   let fsi =
     // on netcore on non-windows we just deflect to fsharpi as usual
-    if Utils.runningOnMono || not FsAutoComplete.Utils.isWindows then Some "fsharpi"
+    if runningOnMono || not isWindows then Some "fsharpi"
     else
       // if running on windows, non-mono we can't yet send paths to the netcore version of fsi.exe so use the one from full-framework
       fsharpInstallationPath |> Option.map (fun root -> root </> "fsi.exe")
 
   let fsc =
-    if Utils.runningOnMono || not FsAutoComplete.Utils.isWindows then Some "fsharpc"
+    if runningOnMono || not isWindows then Some "fsharpc"
```

Comment:
Shouldn't we remove `runningOnMono` here? What if we run mono on windows (yes unlikely but still)?

Reply:
> What if we run mono on windows

That was never supported scenario in FSAC/Ionide - there's whole bunch of other things that won't work in such case